ecommerce/extensions/payment/views.py

```diff
@@ -317,3 +320,85 @@ def get(self, request, *_args, **_kwargs):
         logger.removeHandler(log_handler)
 
         return HttpResponse(output, content_type='text/plain', status=200 if success else 500)
+
+
+class BraintreeCheckoutView(EdxOrderPlacementMixin, View):
+
+    @property
+    def payment_processor(self):
+        return Braintree()
+
+    @method_decorator(transaction.non_atomic_requests)
+    @method_decorator(login_required)
+    @method_decorator(csrf_exempt)
```

Comment:
I did edX like code style for payment views.
I found this comment in one of them
# SS does its own message signing, and their API won't have a cookie value